### ISSUE TITLE
The output of the lstm layer does not align with onnx

### ISSUE THREAD

**Issue by SEMLLYCAT** [Tencent/ncnn#5014]
Dear nihui, I have a question. I hope you could help me @nihui eg:
import torch as nn
class TestLstm(nn.Module):
        def __init__(self, input_size, hidden_size, rnn_type='LSTM', droupt=0, bidirectional=False):
        super(TestLstm, self).__init__()
        self.rnn = nn.LSTM(input_size, hidden_size, 1, dropout=dropout, batch_first=True, bidirectional=False)

def forword(self, x, state_list):
       output, hidden_state = self.rnn(x, state_list)
       return output, hidden_state

if __name__ == "__main__":
       model = TestLstm(input_size=64, hidden_size=64, rnn_type='LSTM', dropout=0, bidirectional=False)
       model.eval()
       x = torch.randn(1, 64, 31, 1)
       state_h0 = torch.randn(1, 31, 64, requires_grad=False)
       state_c0 = torch.randn(1, 31, 64, requires_grad=False)
       state_in = [(state_h0, state_c0)]
       out, state_out = model(x, state_in[0])

onnx2ncnn:
.param
The lstm layer in the.param file is similar：
LSTM                        /lstm_t1/LSTM     3 3 x state_h0 state_c0 /lstm_t1/Transpose_1_out_0 state_out_h0 state_out_c0 0=32 1=8192 2=0

problem：The ncnn inference result is inconsistent with the onnx result, and I cannot align all the results. I don't know whether I used it correctly, and could you please help verify it, thanks

```[tasklist]
### Tasks
```

**Comment by SEMLLYCAT** [comment]
I have solved the problem by adding batch operation, and this issue will be closed. Thanks

**Comment by nihui** [comment]
针对onnx模型转换的各种问题，推荐使用最新的pnnx工具转换到ncnn
In view of various problems in onnx model conversion, it is recommended to use the latest pnnx tool to convert your model to ncnn

```shell
pip install pnnx
pnnx model.onnx inputshape=[1,3,224,224]
```

详细参考文档
Detailed reference documentation
https://github.com/pnnx/pnnx
https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx#how-to-use-pnnx
